hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileUpdateSuite.java

```diff
@@ -184,7 +184,8 @@ public List<HapiSpec> getSpecsInSuite() {
 
     @HapiTest
     private HapiSpec associateHasExpectedSemantics() {
-        return defaultHapiSpec("AssociateHasExpectedSemantics")
+        return propertyPreservingHapiSpec("AssociateHasExpectedSemantics")
+                .preserving("tokens.maxRelsPerInfoQuery")
```

Comment:
Why do we need to change this line?

Reply:
This test changes the property `tokens.maxRelsPerInfoQuery`. We need to revert such changes. Otherwise, other tests can fail.